src/main.js

```diff
@@ -174,6 +174,32 @@ function findIdeaIndex(id) {
   }
 };
 
-function filterIdeas() {
-  
+function filterIdeas(event) {
+  var matchingHTML = '';
+  var searchTerm = event.target.value;
+  for (var i = 0; i < loggedIdea.length; i++) {
+      if (loggedIdea[i].title.includes(searchTerm) || loggedIdea[i].body.includes(searchTerm)) {
+        cardContainer.innerHTML = '';
+        matchingHTML += `
```

Comment:
This looks very similar to `cardTemplate,` we can look at refactoring this together